app/uk/gov/hmrc/ngrdashboardfrontend/config/Module.scala

```diff
@@ -20,8 +20,6 @@ import com.google.inject.AbstractModule
 
 class Module extends AbstractModule {
 
-  override def configure(): Unit = {
-
-    bind(classOf[AppConfig]).asEagerSingleton()
-  }
+  override def configure(): Unit =
```

Comment:
I'd like to discuss this with you please.

Reply:
yeah sure I think is part of will's changes in registration in order to create features